jaeger_client/thrift_gen/zipkincore/ttypes.py

```diff
@@ -5,7 +5,7 @@
 #
 #  options string: py:new_style,tornado
 #
-from past.builtins import xrange
+from six.moves import xrange
```

Comment:
if we generate thrift again, will we run into the same problem again?

Reply:
no, I fixed `thrift-gen-fix.awk`